src/Ledger/PParams.lagda

```diff
@@ -38,43 +41,89 @@ record PoolThresholds : Set where
-        drepThresholds    : DrepThresholds
-        poolThresholds    : PoolThresholds
-        minCCSize         : ℕ
-        ccTermLimit       : ℕ
-        govExpiration     : ℕ
-        govDeposit        : Coin
-        drepDeposit       : Coin
-        drepActivity      : Epoch
-        minimumAVS        : Coin
+  field
+\end{code}
+\emph{Network group}
+\AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
+\begin{code}
+        maxBlockSize       : ℕ
+        maxTxSize          : ℕ
+        maxHeaderSize      : ℕ
+        maxValSize         : ℕ
+        pv                 : ProtVer -- retired, keep for now
+\end{code}
+\emph{Economic group}
+\AgdaTarget{a, b, minUTxOValue, poolDeposit}
+\begin{code}
+        a                  : ℕ
+        b                  : ℕ
+        minUTxOValue       : Coin
+        poolDeposit        : Coin
+\end{code}
+\emph{Technical group}
+\AgdaTarget{Emax, collateralPercent}
+\begin{code}
+        Emax               : Epoch
+        collateralPercent  : ℕ
+\end{code}
+\emph{Governance group}
+\AgdaTarget{drepThresholds, poolThresholds, minCCSize, ccTermLimit, govExpiration, govDeposit, drepDeposit, drepActivity, minimumAVS}
+\begin{code}
+        drepThresholds     : DrepThresholds
+        poolThresholds     : PoolThresholds
+        minCCSize          : ℕ
+        ccTermLimit        : ℕ
+        govExpiration      : ℕ
+        govDeposit         : Coin
+        drepDeposit        : Coin
+        drepActivity       : Epoch
+        minimumAVS         : Coin
 
 paramsWellFormed : PParams → Bool
 paramsWellFormed pp = ⌊ ¬? (0 ∈? setFromList
   (maxBlockSize ∷ maxTxSize ∷ maxHeaderSize ∷ maxValSize ∷ minUTxOValue ∷ poolDeposit
   ∷ collateralPercent ∷ ccTermLimit ∷ govExpiration ∷ govDeposit ∷ drepDeposit ∷ [])) ⌋
   where open PParams pp
 \end{code}
-\caption{Definitions used for protocol parameters}
-\label{fig:defs:pparams}
+\end{AgdaAlign}
+} %% End: small
+\caption{Protocol parameter declarations}
+\label{fig:protocol-parameter-declarations}
 \end{figure*}
+The \AgdaDatatype{ProtVer} type represents the protocol version used in the Cardano ledger.
+It is a pair of natural numbers, the first of which represents the major version, the second
+represents the minor version.
+
+The \AgdaRecord{PParam} type is a record containing parameters used in the Cardano ledger, which we group according
+to the general purpose that each parameter serves. This allows thresholds to be set by group.
+\begin{itemize}
+  \item \AgdaInductiveConstructor{NetworkGroup}: parameters related to the network settings;
+  \item \AgdaInductiveConstructor{EconomicGroup}: parameters related to the economic aspects of the ledger;
+  \item \AgdaInductiveConstructor{TechnicalGroup}: parameters related to technical settings;
+  \item \AgdaInductiveConstructor{GovernanceGroup}: parameters related to governance settings.
+\end{itemize}
+We do not require that each protocol parameter governance action be confined to a single group.
```

Comment:
This paragraph and the sentence above about thresholds should be moved to some other section - this section should be before any section that talk about governance actions, since we need `PParams` to define `GovAction`.

Reply:
Okay, I'll move these remarks to the GovernanceActions module and include them in PR #158.